<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tiger</class>
 <widget class="QDialog" name="Tiger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tiger</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ListQCombobox" name="comboBox_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Try to drag your fasta &lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;alignments&lt;/span&gt; file and drop here if you want to use custom files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="ClickedLableGif" name="label_2">
          <property name="toolTip">
           <string>Brief example</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Ref.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Specify reference sequence in &quot;ti&quot; format (optional)</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_22">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Split file into</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>chunks</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Generate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QSpinBox" name="spinBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Sites will be placed into &lt;int&gt; number of bins. &lt;int&gt; is a whole number. Default is 10. Note: Bin 1 always contains constant sites only.</string>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>bins</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Only include</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="CheckableComboBox" name="comboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>bins</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Exclude</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="CheckableComboBox" name="comboBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>bins</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Unknown characters:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Output format:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Generate rate list</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_40">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Amino acid score matrix to use</string>
          </property>
          <property name="text">
           <string>thread:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Amino acid score matrix to use</string>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Mask characters rather than removing them</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="lineEdit">
          <property name="placeholderText">
           <string>Only put a comma between characters</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>fasta file</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>output bin numbers, sites unsorted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>output bin number, sites sorted on rank</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>displays rank values rather than bin numbers</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>displays rank values and sites sorted on rank FastA</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Run</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1">
      <item row="0" column="0">
       <widget class="ArrowPushButton" name="pushButton">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="../myRes.qrc">
          <normaloff>:/picture/resourses/if_start_60207.png</normaloff>:/picture/resourses/if_start_60207.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="../myRes.qrc">
          <normaloff>:/picture/resourses/if_Delete_1493279.png</normaloff>:/picture/resourses/if_Delete_1493279.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListQCombobox</class>
   <extends>QComboBox</extends>
   <header>src.CustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ArrowPushButton</class>
   <extends>QPushButton</extends>
   <header>src.CustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ClickedLableGif</class>
   <extends>QLabel</extends>
   <header>src.CustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>src.CustomWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../myRes.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox_4</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
